### ISSUE TITLE
BUG: (docs) How do you perform data entry using only keyboard?

### ISSUE THREAD

**Issue by bkidwell** [dbgate/dbgate#254]
**Describe the bug**
*Interface and documentation do not make it clear how to perform data entry operations in a grid, without moving your hands off the keyboard.*

I tried to enter rows by hand using the keyboard, into the "Data" view in DbGate. The Tab key leaves the datagrid and jumps to some other element in the application.

...

On second try just now as I was composing issue, I found through experimentation and tooltips:
* F2 - activate cell editor on current selection
* ESC - finish editing cell
* Arrow keys - move around data grid
* Ctrl-S - save grid edits
* Ins - new row

Aside from some button tooltips, this information can only be discovered by experimentation.

**To Reproduce**
1. Select a table in a database, right click on it and select Open Data.
2. Inspect the data edit screen's buttons and tooltips and try to edit data in the grid using only the keyboard.

Also, I find no keyboard-related information in the online documentation.

**Expected behavior**
The basic keyboard-only editing functionality I discovered by trial-and-error above should be more easily discoverable directly in the data entry form. (For example, a "Keyboard help" button with in-app popup help text.)

More discussion about keyboard navigation should be detailed in a page in the online documentation.

**Version Information (please complete the following information):**
 - OS: Windows 10 Enterprise release 20H2, 64-bit
 - App Version: 4.8.0
 - Install source [e.g. installer/SNAP/Docker/NPM]: Windows .msi
 - Type - Application
 - Database engine: MySQL

*Thank you, creator, for this excellent product!*

**Comment by bkidwell** [comment]
Follow-up: I just found "Keyboard shortcuts" in the "⚙" (gear) menu in the navigation bar. I wouldn't normally think to look in "⚙" for keyboard help. I can't get to Keyboard shortcuts from the textual menu bar.

**Comment by janproch** [comment]
Thanks for useful feedback!

I have fixed data editor, tab key could be now used for navigating between cells.

I have also added Keyboard shortcuts into main menu. However, main purpose of this window is to configure keyboard shortcuts, thats why it is in configuration menu.

From my point of view, primary "documentation" for keyboard shortcuts is context menu, it should contain majority of available commands. Of course, some intuitive keys, like arrows, enter or escape, are not documented this way. There is small minority of keyboard commands, like F2 - edit cell, which is not part of Keyboard shortcuts overview (it cannot be configured). Probably this could be improved a bit.

Also if you press F1, there is command palette - searchable list of currently available commands, also with keyboard shortcuts.

![image](https://user-images.githubusercontent.com/2543891/161074740-dd779420-632f-4ff1-bf29-0d98bc50643b.png)

![image](https://user-images.githubusercontent.com/2543891/161075771-a582b605-f8d3-441b-bdc8-310619d9f05d.png)
